test-app/app/index.html

```diff
@@ -2,23 +2,22 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>amount-input</title>
+    <title>TestApp</title>
```

Comment:
Maybe we can keep the title as it was? WDYT?

```suggestion
    <title>amount-input</title>
```